app/components/Layout/Footer.jsx

```diff
@@ -183,13 +183,17 @@ class Footer extends React.Component {
         return currentNode;
     }
 
-    getNode(node = {url: "", operator: ""}) {
+    getNode(node = {url: "", operator: "", region: ""}) {
         if (!node || !node.url) {
             throw "Node is undefined of has no url";
         }
 
         const {props} = this;
 
+        let testNet;
+        if (node.region) testNet = node.region.startsWith("TESTNET");
```

Comment:
Can you change the check to `url.indexOf("testnet") !== -1` please and remove the overhead again? See `routerTransition._isTestNet`